include/libcyphal/transport/can/delegate.hpp

```diff
@@ -166,6 +206,15 @@ class TransportDelegate
                                                                  const CanardTransferMetadata& metadata,
                                                                  const PayloadFragments        payload_fragments) = 0;
 
+    /// @brief Triggers update of media filters due to a change of an RX subscription.
+    ///
+    /// Actual update will be done on next `run` of transport.
+    ///
+    /// @param is_service `true` if called from a service subscription; `false` for a message one.
+    /// @param is_subscription_added Indicates whether a new subscription was added; otherwise, it was removed.
+    ///
+    virtual void triggerUpdateOfFilters(const bool is_service, const bool is_subscription_added) noexcept = 0;
```

Comment:
Can we please use strong types here instead of the two booleans? At the point of invocation, the two booleans back to back look confusing.

Also, I think I might have mentioned this earlier: `subscription` is not a very good choice of word here. I know that it's coming from libcanard but that's a separate issue. A more appropriate term is "port", can we use that instead?

Reply:
Yeah that's one way to do it. In C++20 we would just introduce a struct but there's no way to sensibly initialize it in the earlier versions of the standard so let's go with an enum I guess.